content/en/docs/concepts/security/hardening-guide/authentication-mechanisms.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Hardening Guide - Authentication Mechanisms
```

Comment:
This page doesn't taste like a concept page.
Maybe it should be a reference page?

Reply:
It's not a reference either. Actual guides really belong in the _tasks_ section. I think that'd be a good home

We could invent a new `content_type`: `guide`. If that feels useful.

Reply:
So, I'm not hugely attached to where this ends up, what I would say is that it's a slightly funny one. The content in tasks feels quite hands-on (i.e. specific commands that are being run) where this is higher level.

Not to say we couldn't put it in tasks, but it's not a 100% fit there, compared to other content in that section. That said, I'd agree isn't not 100% concept either :)

Reply:
Since _tasks_ are for "contains pages that show how to do individual tasks", I don't feel this hardening guide on auth mechanisms belong there unless we extend the scope of the _tasks_ section to include a new content_type of guide

[Good practices for Kubernetes Secrets](https://kubernetes.io/docs/concepts/security/secrets-good-practices/)
and [Role Based Access Control Good Practices](https://kubernetes.io/docs/concepts/security/rbac-good-practices/) are under concepts. I would consider these as guides as well.

There is a Best Practices section, https://kubernetes.io/docs/setup/best-practices/. Is a hardening guide (& other guides) a good fit for the Best Practices section -- I'm leaning towards yes but I can also see this not under best practices.